driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +655,53 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+    import java.io.FileInputStream
+    import java.security.KeyStore
+    import javax.net.ssl.{ KeyManagerFactory, TrustManager }
+
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
In fact I can't restart the jobs in the build referenced below, as I'm not a maintainer, but I did do so on my own forks's Travis and all is passing....
https://travis-ci.org/opetch/ReactiveMongo/builds/185446959